static/js/components/dashboard/CourseAction_test.js

```diff
@@ -117,6 +117,17 @@ describe('CourseAction', () => {
     assertCheckoutButton(elements.button, firstRun.course_id);
   });
 
+  it('hides an invalid date with STATUS_OFFERED', () => {
+    let course = findCourse(course => (
+      course.runs.length > 0 &&
+      course.runs[0].status === STATUS_OFFERED
+    ));
+    alterFirstRun(course, {enrollment_start_date: '1999-13-92'});
```

Comment:
`alterFirstRun` will modify `course` so you'll need to clone it first. Can you change `findCourse` to `findAndCloneCourse` above?